05-how-to-contribute-to-open-source.md

```diff
@@ -214,35 +200,21 @@ You'll need to have these tools donwloaded and installed on your local machine:
 
    > **Tip**: If you don't see your profile in the "Contributors" section, zoom out the screen until you can see it before taking a screenshot.
```

Comment:
docsify supports important and note [helpers](https://docsify.js.org/#/helpers). And we want to start applying these where we see fit.

Please update this to:

```suggestion
   !> If you don't see your profile in the "Contributors" section, zoom out the screen until you can see it before taking a screenshot.
```

Reply:
Done